app/assets/javascripts/sign_up_async_request.js.erb

```diff
@@ -10,6 +10,7 @@ $( document ).ready(function() {
 			ajaxRequestArray.pop();
 		}
 
+		$(this).attr('validation-passed',false);
```

Comment:
A linter would recommend adding a space after the comma.

Reply:
Fixed.